scripts/release.sh

```diff
@@ -89,22 +93,62 @@ function tag_images() {
     make release RELEASE_ARGS="$images --tag ${release['version']}"
 }
 
+### Functions: Branch Stage ###
+
+function update_shipyard_base() {
+    sed -i -E "s/(shipyard-dapper-base):.*/\1:${release['branch']}/" projects/${project}/Dockerfile.dapper
+}
+
+function adjust_shipyard() {
+    local project=shipyard
+
+    sed -e "s/devel/${branch}/" -i projects/shipyard/Makefile.versions
```

Comment:
nit:  shall we tune it to be more specific?

https://github.com/submariner-io/shipyard/blob/e6b9eb078e2ffc17ace6e2437056e420c5c3c0ab/Makefile.versions

Not super risky I see because that makefile is quite minimalist.

Reply:
As you said the Makefile is very small so I don't see a need to do that ATM.
We can always change it in the future